requirements_test_all.txt

```diff
@@ -2557,6 +2557,15 @@ volvocarsapi==0.4.3
 # homeassistant.components.verisure
 vsure==2.6.7
 
+# homeassistant.components.vasttrafik
+vtjp==0.2.1
+
+# homeassistant.components.vulcan
+vulcan-api==2.4.2
+
+# homeassistant.components.vultr
+vultr==0.1.2
+
```

Comment:
Why are these added? Is it a merge mistake?

Reply:
Oh yeah, this seems like a merge mistake that I have overlooked. Sorry, I will remove those.